### PR TITLE
CSS project

### PR DESCRIPTION
**Pull request by Deepanshu-Singh45**

### CODE REVIEW

File: css assignment/gallery.html [hunk 1]
```diff
@@ -16,54 +16,50 @@
         <a href="./gallery.html" class="items"> My Gallery </a>
         
     </nav>
+    <h1 class="heading" style="text-align:center;">Gallery</h1>
 
-    <div class="container">
-      <h1 class="heading">Gallery</h1>
-      <ul>
-        <li>
+    <div class="flex-container">
```
Comment (ShridharDaga) [inline css assignment/gallery.html]: can you make all gallary img content to the center

now it is more on left side

File: css assignment/styles.css [hunk 2]
```diff
@@ -13,11 +13,11 @@ nav {
   position: sticky;
```
Comment (ShridharDaga) [inline css assignment/styles.css]: make navbar content to center
now it is moved on top end

File: css assignment/aboutme.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (ShridharDaga) [inline css assignment/aboutme.html:2]: make contetnt to center of screen add some padding margin and width 
sorry not for this page

but contact me page

File: css assignment/gallery.html (hunk 1 above)
Comment (Deepanshu-Singh45) [reply css assignment/gallery.html]: Done.

File: css assignment/styles.css (hunk 2 above)
Comment (Deepanshu-Singh45) [reply css assignment/styles.css]: Done

File: css assignment/aboutme.html (hunk 3 above)
Comment (Deepanshu-Singh45) [reply css assignment/aboutme.html:2]: Done

**Review by ShridharDaga — APPROVED** [review]

**Pull request merged**